packages/dx-react-grid/docs/reference/paging-panel.md

```diff
@@ -44,9 +44,10 @@ totalPages | number | Specifies the total page count.
 currentPage | number | Specifies the current page.
 onCurrentPageChange | (page: number) => void | Handles the current page changes.
 pageSize | number | Specifies the page size.
+totalCount | number | Specifies the total row count.
 onPageSizeChange | (size: number) => void | Handles the page size changes.
 pageSizes | Array&lt;number&gt; | Specifies the page sizes that a user can select.
-getMessage | ([messageKey](#localization-messages): string) => string | Returns the paging panel's text.
+getMessage | ([messageKey](#localization-messages): string, params?: { from: number, to: number, count: number }) => string | Returns the paging panel's text.
```

Comment:
Usually, we [use](https://github.com/DevExpress/devextreme-reactive/blame/9f9230074f5c41847a9a1553f5f0cbeff94254f3/packages/dx-react-grid/docs/reference/table-header-row.md#L62) `parameters` instead of `params`.